pyfolio/pos.py

```diff
@@ -171,3 +173,35 @@ def get_turnover(transactions, positions, period=None):
     turnover = traded_value / 2.0
     turnover_rate = turnover / portfolio_value
     return turnover_rate
+
+
+def get_sector_exposures(positions, symbol_sector_map):
+    """
+    Sum position exposures by sector.
+
+    Parameters
+    ----------
+    positions : pd.DataFrame
+        Contains position values or amounts.
+    symbol_sector_map : dict
+        Sector to security identifier mapping.
+        Sectors as keys, security ids as values.
```

Comment:
I don't think this description is correct if the intent here is to be summing by sector.  The values of your grouping dict should be your sectors, not the keys:

```
In [8]: df = DataFrame(arange(10).reshape(5, 2))

In [9]: df
Out[9]:
   0  1
0  0  1
1  2  3
2  4  5
3  6  7
4  8  9

In [10]: df = DataFrame(arange(10).reshape(5, 2), columns=['x', 'y'], index=['a', 'b', 'c', 'd', 'e'])

In [11]: df
Out[11]:
   x  y
a  0  1
b  2  3
c  4  5
d  6  7
e  8  9

In [12]: grouper = {'a': 'group_0', 'b': 'group_1', 'c': 'group_0', 'd': 'group_1', 'e': 'group_2'}

In [13]: df.groupby(by=grouper).sum()
Out[13]:
         x   y
group_0  4   6
group_1  8  10
group_2  8   9
```